graphics/nanovg/Portfile

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+set shorthash       2bead03
+set fullhash        2bead03bea43b2418060aaa154f972829995e663
+set lastcommit      20200320
+github.setup        memononen nanovg $shorthash
+version             $lastcommit
+categories          graphics
+platforms           darwin
+license             zlib
+maintainers         @jasonliu-- openmaintainer
+homepage            ${github.homepage}
+description         antialiased 2-D vector drawing library on top of \
+                    OpenGL for UI and visualizations
+long_description    NanoVG is a small antialiased vector graphics \
+                    rendering library for OpenGL. It has a lean API \
+                    modeled after the HTML5 <canvas> API. It is \
+                    intended to be a practical toolset for building \
+                    scalable user interfaces and visualizations.
+
+checksums           rmd160  b2b619156f4321b67df34e3ca8aab41e972c8328 \
+                    sha256  6467b3274d148ed550b2f148dfad0c342fa948665b15b5f3e662ae17a04443c4 \
+                    size    2032399
```

Comment:
Please put the settings ( GitHub setup, descriptions, checksums etc. ) specific to the primary port, and not the subplot, inside a

```
if {$subport eq "nanovg"} {
 ....
}
```

block, just to avoid them for the sub port being first set then over-ridden later on.

Reply:
Okay, I'll make the changes now. Since this is my first time making something with a subport, I've been the portfile for `VLC` as a reference. You'll have to let me know whether the settings that I'm wrapping looks correct.